bpf/http_ssl.h

```diff
@@ -92,7 +92,8 @@ static __always_inline void https_buffer_event(void *buf, int len, connection_in
         }
 
         if (packet_type == PACKET_TYPE_REQUEST) {
-            process_http_request(info, buf);
+            process_http_request(info);
+            bpf_memcpy(info->buf, buf, FULL_BUF_SIZE);
```

Comment:
The only part that needed to copy the request buffer was from the SSL path, we do it here now to avoid duplicate buffer copies in the main path.